bedrock/mozorg/tests/test_helper_misc.py

```diff
@@ -506,6 +506,69 @@ def test_high_res_img_with_l10n_and_optional_attributes(self):
         self.assertEqual(markup, expected)
 
 
+@override_settings(STATIC_URL="/media/")
+class TestLazyImg(TestCase):
```

Comment:
**Change required**
As above, this code is no longer in `master`: https://github.com/mozilla/bedrock/commit/1c4b333359bdb959ae6a0eb289ad8c4d25fa5050